docs/MigrationGuide_to_4.0.md

```diff
@@ -139,7 +178,7 @@ struct icalrecurrencetype {
 Memory is allocated in the required size using the new `icalrecur_resize_by()` function. It is
 automatically freed together with the containing `icalrecurrencetype`. As the size of the array is
 stored explicitly, no termination of the array with special value `ICAL_RECURRENCE_ARRAY_MAX` is
-required anymore.
+required anymore.  Iterating the array by comparing the iterator to the `size` member value.

```

Comment:
Maybe 'The array is iterated by comparing ...'

Reply:
done